test/instances/router.lua

```diff
@@ -13,6 +13,7 @@ _G.vshard = {
     router = require('vshard.router'),
 }
 _G.ivshard = _G.vshard
+_G.ifiber = require('fiber')
```

Comment:
I am going to make a moderately harsh review at first. Let me know if it is too complicated and you would prefer a light version after pushing which I will make some review fixes myself.

Here, please, put the require a bit higher. I am trying to keep them alphabetically sorted in this file and all new ones. It is already ruined in most of the older files, but here I want to make it right.